charts/datadog/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Datadog changelog
 
+## 3.74.2
+
+* Mount /usr/lib/sysimage/rpm when using host SBOM feature.
```

Comment:
```suggestion
* Mount `/usr/lib/sysimage/rpm` in the Agent DaemonSet when using host SBOM feature (required on hosts running Amazon Linux distributions).
```
Could you make the release note slightly more explicit (possibly referencing which distros/versions make use of this directory) ?